lib/src/base-styles/_nx-variables.scss

```diff
@@ -48,6 +48,11 @@
 
   --nx-box-shadow-focus: 0 0 3px 1px var(--nx-color-interactive-shadow-focus);
 
+  // for dropdown 
+  --nx-box-shadow-dropdown: 0 2px 12px 0 rgba(74, 77, 113, 0.2),
```

Comment:
This should be added to the docs in `CssVariablesPage.tsx`

Reply:
Updated here dd768293c1a6c103672421cd3b35ebce9a7d31fb.